recipe/build.sh

```diff
@@ -30,10 +30,10 @@ RELEASE_DIR=$PYPY3_SRC_DIR/pypy/tool/release
 PYPY_PKG_NAME=pypy3
 BUILD_DIR=${PREFIX}/../build
 TARGET_DIR=${PREFIX}/../target
-ARCHIVE_NAME="${PYPY_PKG_NAME}-${PKG_VERSION}"
-PYPY_PACKAGE_WITHOUTTK=1
-PYPY_NO_EMBED_DEPENDENCIES=1
-PYPY_NO_MAKE_PORTABLE=1
+export ARCHIVE_NAME="${PYPY_PKG_NAME}-${PKG_VERSION}"
+export PYPY_PACKAGE_WITHOUTTK=1
+export PYPY_NO_EMBED_DEPENDENCIES=1
```

Comment:
If this wasn't working before then this PR seems like a fairly important fix?

Reply:
I'm not sure. I can never tell when I am required to use `export` in a bash script that calls other executables. These environment variables are used int these lines
`PYTHONPATH=../.. ./${PYPY_PKG_NAME}*-c ../../lib_pypy/pypy_tools/build_cffi_imports.py`
and 
`${PYTHON} ./package.py --targetdir="$TARGET_DIR" --archive-name="$ARCHIVE_NAME"` which internally use `subprocess.Popen`.